src/test/java/com/epam/tc/hw6/DriverFabric.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.tc.hw6;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.Locale;
+import lombok.NonNull;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class DriverFabric  {
+
+    public static WebDriver getWebDriver(final boolean isLocal, final String hub, @NonNull final String browser) {
```

Comment:
Parameters final boolean isLocal and final String hub are never used...

Reply:
I used the suggestions of the lecturer from the previous course which you shared with me. The framework has major changes..